### PR TITLE
Run Databricks IT with python-xdist parallel

### PR DESCRIPTION
**Pull request by NvTimLiu**
Enable the databricks IT tests to run in parallel, issue: https://github.com/NVIDIA/spark-rapids/issues/1499

Signed-off-by: Tim Liu <timl@nvidia.com>

### CODE REVIEW

File: jenkins/databricks/build.sh
Comment (jlowe) [inline jenkins/databricks/build.sh:107]: 2021 copyrights need to be added to this file

File: jenkins/databricks/build.sh
Comment (jlowe) [inline jenkins/databricks/build.sh:131]: Is this supposed to be `./run_pyspark_from_build.sh`?  This script isn't designed to be sourced, and it's confusing to see it treated that way.  I'd rather see it executed directly or used via something like `bash -c`.

**Review by jlowe — CHANGES_REQUESTED** [review]

File: jenkins/databricks/build.sh
Comment (NvTimLiu) [reply jenkins/databricks/build.sh]: changed to `bash run_pyspark_from_build.sh --runtime_env="databricks"`

File: jenkins/databricks/build.sh
Comment (NvTimLiu) [reply jenkins/databricks/build.sh:107]: fixed

**Comment by tgravescs** [comment]
how long was build taking with this change?

**Comment by tgravescs** [comment]
never mind saw your post on the issue

**Review by jlowe — APPROVED** [review]

**Comment by tgravescs** [comment]
lets not merge until all tests pass -> see the issue

**Comment by jlowe** [comment]
Converted this to a draft until it's ready to be merged.

**Review by gerashegalov — APPROVED** [review]
LGTM

**Comment by tgravescs** [comment]
@NvTimLiu are you going ot update this based on your finding?  we can skip the tests  in the issue https://github.com/NVIDIA/spark-rapids/issues/1563 for now

**Comment by tgravescs** [comment]
replaced by https://github.com/NVIDIA/spark-rapids/pull/1645

**Pull request closed without merging**
